src/H5Fmodule.h

```diff
@@ -31,8 +31,34 @@
 
 /**
  * \defgroup H5F H5F
- * \brief File Interface
- * \todo Describe concisely what the functions in this module are about.
+ *
+ * Use the functions in this module to manage HDF5 files.
+ *
+ * In the code snippets below, we show the skeletal life cycle of an HDF5 file,
+ * when creating a new file (left) or when opening an existing file (right).
+ * File creation is essentially controlled through \ref FCPL, and file access to
+ * new and existing files is controlled through \ref FAPL. The file \c name and
```

Comment:
The use of CAP and no cap for variable names is inconsistent.

Reply:
These (FCPL, FAPL) are NOT variable names.